### PR TITLE
add runbook for outdated vmi workloads alert

### PR DESCRIPTION
**Pull request by davidvossel**
This handles the `OutdatedVirtualMachineInstanceWorkloads` alert

### CODE REVIEW

File: runbooks/OutdatedVirtualMachineInstanceWorkloads.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  imagePullPolicy: IfNotPresent
+  workloadUpdateStrategy:
+    workloadUpdateMethods:
+      - LiveMigrate
+      - Evict
+    batchEvictSize: 10
+    batchEvictInterval: "1m"
+```
+
+If automated workload updates are not enabled, consider enabling them. This will cause the KubeVirt control place to automatically update the VMI workloads using the methods defined in the `workloadUpdateMethods` field.
```
Comment (sradco) [inline runbooks/OutdatedVirtualMachineInstanceWorkloads.md:43]: @davidvossel  typo, place instead of plane.

**Comment by sradco** [comment]
@davidvossel  Hi, Thank you for your patch. Can you please ass a signoff.
Also, will you be able to add some commands that the user can run to diagnose and mitigate the issue?

File: runbooks/OutdatedVirtualMachineInstanceWorkloads.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+```bash
+kubectl get vmi -l “kubevirt.io/outdatedLauncherImage” --all-namespaces
+```
+
+## Mitigation
+
+### Check if automated workload updates are enabled
+
+Check the KubeVirt CR used to install KubeVirt to see if the `workloadUpdateStrategy` is configured on the KubeVirt CR's spec.
```
Comment (erkanerol) [inline runbooks/OutdatedVirtualMachineInstanceWorkloads.md:23]: What about adding a link to https://kubevirt.io/user-guide/operations/updating_and_deletion/#updating-kubevirt-workloads ?

File: runbooks/OutdatedVirtualMachineInstanceWorkloads.md (hunk 1 above)
Comment (davidvossel) [reply runbooks/OutdatedVirtualMachineInstanceWorkloads.md]: fixed!

File: runbooks/OutdatedVirtualMachineInstanceWorkloads.md (hunk 2 above)
Comment (davidvossel) [reply runbooks/OutdatedVirtualMachineInstanceWorkloads.md:23]: fixed

**Pull request merged**